ios/Plugin/Plugin.swift

```diff
@@ -198,6 +200,127 @@ public class NativeBiometric: CAPPlugin {
         }
     }
     
+    @objc func getPublicKey(_ call: CAPPluginCall){
+        var obj = JSObject()
+        
+        do {
+            let publicKeyFromKeychain = try getPublicFromKeychain()
+            var error:Unmanaged<CFError>?
+            if let cfdata = SecKeyCopyExternalRepresentation(publicKeyFromKeychain, &error) {
+               let data:Data = cfdata as Data
+               let b64Key = data.base64EncodedString()
+                obj["publicKey"] = b64Key
+                call.resolve(obj)
+            }
+        } catch {
+            do{
+                let generatedPublicKey = try generatePublicKey()
+                try storePublicKeyToKeychain(generatedPublicKey)
+                
+                var error:Unmanaged<CFError>?
+                if let cfdata = SecKeyCopyExternalRepresentation(generatedPublicKey, &error) {
+                   let data:Data = cfdata as Data
+                    let b64Key = data.base64EncodedString()
+                    obj["publicKey"] = b64Key
+                    call.resolve(obj)
+                }
+            }catch{
+                call.reject("Cannot generate public key")
+            }
+        }
+    }
+    
+    @objc func signData(_ call: CAPPluginCall){
+        var obj = JSObject()
+        
+        let query: [String: Any] = [
+            kSecClass as String: kSecClassKey,
+            kSecAttrKeyClass as String: kSecAttrKeyClassPrivate,
+            kSecAttrKeyType as String: kSecAttrKeyTypeRSA,
+            kSecAttrKeySizeInBits as String: 2048,
+            kSecAttrApplicationTag as String: privateKeyTag,
+            kSecReturnRef as String: true
+        ]
+        
+        var item: CFTypeRef?
+        let status = SecItemCopyMatching(query as CFDictionary, &item)
+        guard status == errSecSuccess else { return }
+        let privateKey = item as! SecKey
+        
+        guard let challengeString = call.getString("challengeString") else {
+            call.reject("No challenge string was provided")
+            return
+        }
+        
+        guard let messageData = challengeString.data(using: String.Encoding.utf8) else {
+            call.reject("Invalid message to sign")
+            return
+        }
+        
+        do{
+            guard let signData = SecKeyCreateSignature(
+            privateKey,
+            SecKeyAlgorithm.ecdsaSignatureMessageX962SHA256,
```

Comment:
```suggestion
            SecKeyAlgorithm.rsaSignatureMessagePKCS1v15SHA256,
```

IIRC in android we're using `SHA256withRSA` [means the signature is using algorithm `RSA` that implements `PKCS v1.5` and hash the data with `SHA256`](https://stackoverflow.com/questions/21018355/sha256withrsa-what-does-it-do-and-in-what-order)

So I think we should use [`rsaSignatureMessagePKCS1v15SHA256`](https://developer.apple.com/documentation/security/seckeyalgorithm/1644047-rsasignaturemessagepkcs1v15sha25)